src/lib.rs

```diff
@@ -81,21 +81,22 @@
 //! ❌ = will not compile
 //!
 //! ## `Notification`
-//! | method              | XDG   | macOS | windows |
-//! |---------------------|-------|-------|---------|
-//! |  `fn appname(...)`  |  ✔︎    |       |        |
-//! |  `fn summary(...)`  |  ✔︎    | ✔︎     |  ✔︎    |
-//! |  `fn subtitle(...)` |       | ✔︎     |  ✔︎    |
-//! |  `fn body(...)`     |  ✔︎    | ✔︎     |  ✔︎    |
-//! |  `fn icon(...)`     |  ✔︎    |       |        |
-//! |  `fn auto_icon(...)`|  ✔︎    |       |        |
-//! |  `fn hint(...)`     |  ✔︎    | ❌    | ❌    |
-//! |  `fn timeout(...)`  |  ✔︎    |       |  ✔︎    |
-//! |  `fn urgency(...)`  |  ✔︎    | ❌    | ❌    |
-//! |  `fn action(...)`   |  ✔︎    |       |        |
-//! |  `fn id(...)`       |  ✔︎    |       |        |
-//! |  `fn finalize(...)` |  ✔︎    | ✔︎     |  ✔︎    |
-//! |  `fn show(...)`     |  ✔︎    | ✔︎     |  ✔︎    |
+//! | method                | XDG   | macOS | windows |
+//! |-----------------------|-------|-------|---------|
+//! |  `fn appname(...)`    |  ✔︎    |       |        |
+//! |  `fn summary(...)`    |  ✔︎    | ✔︎     |  ✔︎    |
+//! |  `fn subtitle(...)`   |       | ✔︎     |  ✔︎    |
+//! |  `fn body(...)`       |  ✔︎    | ✔︎     |  ✔︎    |
+//! |  `fn icon(...)`       |  ✔︎    |       |        |
+//! |  `fn auto_icon(...)`  |  ✔︎    |       |        |
+//! |  `fn hint(...)`       |  ✔︎    | ❌    | ❌    |
+//! |  `fn timeout(...)`    |  ✔︎    |       |  ✔︎    |
+//! |  `fn urgency(...)`    |  ✔︎    | ❌    | ❌    |
+//! |  `fn action(...)`     |  ✔︎    |       |        |
+//! |  `fn id(...)`         |  ✔︎    |       |        |
+//! |  `fn finalize(...)`   |  ✔︎    | ✔︎     |  ✔︎    |
+//! |  `fn show(...)`       |  ✔︎    | ✔︎     |  ✔︎    |
+//! |  `fn show_async(...)` |  ✔︎    | ✔︎     |  ❌    |
```

Comment:
This file is best viewed with whitespace hidden - this line is the only real change